build_test/mono_pkg.yaml

```diff
@@ -7,7 +7,7 @@ stages:
     - dartfmt: sdk
     - dartanalyzer: --fatal-infos --fatal-warnings .
 - unit_test:
-  - test: --test-randomize-ordering-seed=random
+  - command: dart --no-sound-null-safety test --test-randomize-ordering-seed=random
```

Comment:
We shouldn't need this if the tests themselves are opted out (this should be the default for opted out tests).